From 1477bce780cd693614c5714103e84f6c1897b2c7 Mon Sep 17 00:00:00 2001
From: Arnaud Ferraris <arnaud.ferraris@collabora.com>
Date: Fri, 23 Oct 2020 13:40:44 +0200
Subject: [PATCH] shell: honor configured rotation

Resetting the default transform causes the user-configured rotation to
be ignored. In cases where the transform needs to be reset, this is
already done in lockscreen manager.
---
 src/shell.c | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/src/shell.c b/src/shell.c
index f38bdab4..48dd4c2b 100644
--- a/src/shell.c
+++ b/src/shell.c
@@ -484,10 +484,6 @@ setup_idle_cb (PhoshShell *self)
 
   phosh_session_register (PHOSH_APP_ID);
 
-  /* If we start rotated, fix this up */
-  if (phosh_shell_get_transform (self) != PHOSH_MONITOR_TRANSFORM_NORMAL)
-    phosh_shell_set_transform (self, PHOSH_MONITOR_TRANSFORM_NORMAL);
-
   priv->startup_finished = TRUE;
 
   return FALSE;
-- 
2.26.2

